src/components/form/dt-text/dt-text.js

```diff
@@ -101,15 +108,15 @@ export class DtTextField extends DtFormBase {
           id="${this.id}"
           name="${this.name}"
           aria-label="${this.label}"
-          type="text"
+          type="${this.type}"
```

Comment:
There won't be an initial value so this should have a fallback to text.
${this.type || 'text'}

Reply:
@cairocoder01 Done!